### DISCUSSION TITLE
frontend setup - using devcontainer

### DISCUSSION THREAD

**Discussion by Santoshactive147** [inventree/InvenTree#8765] [Q&A]
![Screenshot (2)](https://github.com/user-attachments/assets/1d25b756-4f85-4bbd-bd7a-5bc58ceac92d)
![Screenshot (1)](https://github.com/user-attachments/assets/1f36e8a7-2c24-4cdf-9cdd-159d8d1fa643)

Can Anyone help how do i solve this 
please explain step wise , thank you !

**Comment by wolflu05** [comment]
Please provide the exact step you are not understanding. The guid you have open in the first screenshot already describes pretty good how to do it.

@matmair we may have forgoten to change the path in step 4

**Comment by Santoshactive147** [comment]
@wolflu05 
  "mounts": [
  "source=,target=/workspaces/InvenTree,type=bind,consistency=cached"
],

like which path i have to enter in source = ""
what  is inventree-plugins and where i will get this ?

> help me regarding folder structure of workspace and inventree -plugin where do i create ?
thanks @wolflu05

**Reply by wolflu05** [reply]
The path where you plugin repository is stored on your local disk/system. E.g. if you clone inventree to `/home/wolflu/Development/InvenTree` and your plugin repo to `/home/wolflu/Development/inventree-bulk-plugin`. You enter `source=/home/wolflu/Development/inventree-bulk-plugin,target=/workspaces/inventree-bulk-plugin,...` .

**Reply by Santoshactive147** [reply]
@wolflu05 thanks buddy !
